config

```diff
@@ -11,3 +11,7 @@ path /static2 file_handler {
 path /status StatusHandler {}
 
 default NotFoundHandler {}
+
+path /proxy ProxyHandler {
+    root www.ucla.edu;
```

Comment:
How is the remote port found if one is to be passed in? I don't see it in the config file. Is it located somewhere else?